manifest.json

```diff
@@ -1,6 +1,21 @@
 {
-    "name": "Hello, world!",
+    "name": "Totally for real actual ghostery extension",
     "version":"0.000001",
-    "manifest_version": 2
+    "manifest_version": 2,
```

Comment:
by convention  `manifest_version` goes at the very top

Reply:
Manifest version put on the top of manifest.json